frontend/js/components/HistoryTable/HistoryTableRow.component.js

```diff
@@ -0,0 +1,29 @@
+class HistoryTableRow extends HTMLElement {
+  constructor() {
```

Comment:
follow the order of the function placement as the previous file

Reply:
HistoryRow component is deleted as of now.